internal/cli/config/prompt.go

```diff
@@ -0,0 +1,105 @@
+package config
+
+import (
+	"github.com/AlecAivazis/survey/v2"
+	"github.com/mongodb/mongocli/internal/config"
+	"github.com/mongodb/mongocli/internal/validate"
+)
+
+const (
+	omBaseURLHelp = "FQDN and port number of the Ops Manager Application."
+	projectHelp   = "ID of an existing project that your API keys have access to. If you don't enter an ID, you must use --projectId for every command that requires it."
+	orgHelp       = "ID of an existing organization that your API keys have access to. If you don't enter an ID, you must use --orgId for every command that requires it."
+	atlasAPIHelp  = "Please provide your API keys. To create new keys, see the documentation: https://docs.atlas.mongodb.com/configure-api-access/"
+	omAPIHelp     = "Please provide your API keys. To create new keys, see the documentation: https://docs.opsmanager.mongodb.com/current/tutorial/configure-public-api-access/"
+)
+
+func newOMURLInput() survey.Prompt {
+	return &survey.Input{
+		Message: "URL to Access Ops Manager:",
+		Help:    omBaseURLHelp,
+		Default: config.OpsManagerURL(),
+	}
+}
+
+func newOrgIDInput() survey.Prompt {
+	return &survey.Input{
+		Message: "Default Org ID:",
+		Help:    orgHelp,
+		Default: config.OrgID(),
+	}
+}
+
+func newOrgSelect(a []string) survey.Prompt {
```

Comment:
[np] `a` -> `orgs` or `options`?

Reply:
I'm a lazy bastard who embraced the go philosophy of single letter variables, usually slices are just `a` or `s` but I'll change this to options

Reply:
:D